docs/config/dev-server.mdx

```diff
@@ -0,0 +1,143 @@
+import DevServerExample from "@site/docs/config/_partials/examples/_dev-server-example.mdx";
+
+# devServer
+
+С помощью секции `devServer` можно запускать сервер, на который будут ходить тесты. Запуск происходит во время инициализации Testplane на событие INIT.
+
+## Пример использования {#example}
+
+<DevServerExample />
+
+## Справка по секции devServer
+
+<table>
+    <thead>
+    <tr>
+        <td>**Параметр**</td>
+        <td>**Тип**</td>
+        <td>**По&nbsp;умолчанию**</td>
```

Comment:
why you use `&nbsp` here? Can't we just use simple space?

Reply:
Copied it from the original docs... I guess it's used here to prevent these words to be split into two lines.